src/App.tsx

```diff
@@ -2,108 +2,39 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { CommentsFromServer } from './types/Comments';
+import { PostFromServer, Posts } from './types/Posts';
+import { UsersFromServer } from './types/Users';
+import { PostList } from './components/PostList';
+
+function getPostById(
```

Comment:
if you return an array of objects its not really getPost it's more like getPosts